app/config/schema/config.proto

```diff
@@ -13,9 +13,14 @@ message Config {
     // Optional. Defaults to false.
     bool debug = 1;
 
+    // Title for main page
+    // document.title in the DOM
+    // Required.
+    string pageTitle = 18;
```

Comment:
Super nit: this should technically be `page_title` according to proto style. The json parser will still accept `pageTitle` in the config files.